cibyl/utils/sorting.py

```diff
@@ -86,8 +81,8 @@ class BubbleSortAlgorithm(SortingAlgorithm):
     """
 
     @overrides
-    def sort(self, iterable):
-        def compare(lft, rght):
+    def sort(self, iterable: Iterable) -> list:
+        def compare(lft: Any, rght: Any):
```

Comment:
We should know when to be less demanding. Type hinting an inner function does not have much value, even less if the types are undefined. Nobody is going to be able to see it from the outside.